### PR TITLE
FEAT: Added single video page

### PR DESCRIPTION
**Pull request by zoyatanweer**
Added single video page. Click on the video card image to open single video plater page.

Files to check:
-pages/SingleVideo/SingleVideo.jsx
-pages/SingleVideo/SingleVideo.css
-components/VideoCard/VideoCard.jsx
-App.js

### CODE REVIEW

**Review by srishtigautam21 — APPROVED** [review]
The code lis clean and looks good to merge

File: src/pages/Homepage/Homepage.css [hunk 1]
```diff
@@ -142,6 +142,7 @@
 
 .main-content {
   background-color: #343a40;
+  /* background-color: #212529; */
   margin-left: 1rem;
   /* margin-left: 0; */
   /* padding: 2rem; */
```
Comment (Jia-A) [inline src/pages/Homepage/Homepage.css:148]: Remove this commented code if not needed later.

**Review by Jia-A — APPROVED** [review]
Everything looks good, you can merge it.

**Review by dev-prabhat — APPROVED** [review]
Looks good to me

**Pull request merged**